entity-service/src/main/kotlin/com/egm/stellio/entity/repository/Neo4jRepository.kt

```diff
@@ -475,14 +475,22 @@ class Neo4jRepository(
             .first()
     }
 
-    fun filterExistingEntitiesIds(entitiesIds: List<String>): List<String> {
+    fun filterExistingEntitiesAsIds(entitiesIds: List<String>): List<String> {
+        val transformToId = { map: Map<String, Any> -> map["id"] as String }
+        return filterExistingEntitiesIdsAs(entitiesIds, transformToId)
+    }
+
+    private fun <T> filterExistingEntitiesIdsAs(
+        entitiesIds: List<String>,
+        transformFunc: (Map<String, Any>) -> T
```

Comment:
is it really useful to pass a transformation function, since this method is only called by `filterExistingEntitiesAsIds` (with always the same transformation method)?

Reply:
You're right, i'll remove it